SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +156,20 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// Alternative connection method, when identifier and services are known for a peripheral.
+    ///
+    /// - Parameter peripheralUUID: The unique identifier (UUID) for the given peripheral.
+    /// - Parameter serviceUUIDs: The service UUIDs to for the given peripheral.
+    /// - Parameter timeout: The time in seconds before the connection attempt is stopped and the completion closure is called with a failed result.
+    /// - Parameter completion: The closures, called upon completion (succesful or otherwise) of the connection.
+    public func connect(peripheralUUID: UUID,
```

Comment:
There are still some liberties taken... As you can see, the Central class is a Singleton, therefore I felt it was cleaner to put the Connect method inside the Peripheral interface. peripheral.connect(:) looks better than Central.sharedInstance.connect(peripheral:completion:), there is no point in having the connect method in the Central when there is only one central instance anyway. Do still note that having the connect method in Peripheral is still not adding any functionalities that the CoreBluetooth APIs already provide.